test/utils/utils.go

```diff
@@ -383,7 +379,10 @@ func tagOutputToMap(imagesOutput []string) map[string]string {
 		if len(tmp) < 2 {
 			continue
 		}
-		m[tmp[0]] = tmp[1]
+		if m[tmp[0]] == nil {
```

Comment:
WARNING: please review carefully, because I don't speak Golang, and there is almost certainly a more idiomatic way to do this.